From 6b58257da0c2346e213ecf4ae80c31340d0f9638 Mon Sep 17 00:00:00 2001
From: Markus Pargmann <mpa@pengutronix.de>
Date: Fri, 5 Jul 2013 15:45:09 +0200
Subject: [PATCH 050/132] ARM: dts: imx27 phyCARD-S remove wrong I2C RTC

Fixup of commit "ARM: dts: Add device tree support for phycard pca100".

Remove wrong I2C RTC node.

Signed-off-by: Markus Pargmann <mpa@pengutronix.de>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts b/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
index ec52b6d..01ee807 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycard-s-som.dts
@@ -55,11 +55,6 @@
 		reg = <0x4a>;
 	};
 
-	rtc@51 {
-		compatible = "nxp,pcf8563";
-		reg = <0x51>;
-	};
-
 	at24@52 {
 		compatible = "at,24c32";
 		pagesize = <32>;
-- 
1.7.10.4

